frontend/fingertips-frontend/components/atoms/BenchmarkTooltipArea/index.tsx

```diff
@@ -28,10 +28,25 @@ export function BenchmarkTooltipArea({
   measurementUnit,
   tooltipType,
 }: Readonly<BenchmarkTooltipArea>) {
+  const indicatorDataForAreaForMostRecentYear =
+    getIndicatorDataForAreasForMostRecentYearOnly([indicatorData]);
```

Comment:
Since this is just for a single areas indicator data. Can we use this function instead `sortHealthDataPointsByDescendingYear` 

Hopefully that may need the need to get the first element of the array underneath.